src/pint/toa.py

```diff
@@ -771,7 +771,7 @@ def read_toa_file(filename, process_includes=True, cdict=None, dir=None):
                 cdict[cmd] = float(d["Command"][1]) * u.MHz
             elif cmd in ("EFAC", "PHA1", "PHA2"):
                 cdict[cmd] = float(d["Command"][1])
-                if cmd in ("PHA1", "PHA2", "TIME", "PHASE"):
```

Comment:
These values are not reachable.

Reply:
TIME and PHASE commands are treated a few lines above here. I think it is OK to remove this.